test_fixtures/coverage/functional_test/tool/selenium-server

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+exec java -jar $DIR/selenium-server-standalone.jar "$@"
```

Comment:
I committed a local `selenium-server` executable along with the JAR (below) so that we can get more consistent results on this integration test, and also just to prove that the `SeleniumHelper` class works with a globally installed selenium server or with a local executable.